GVFS/GVFS.Installer.Mac/scripts/postinstall

```diff
@@ -1,4 +1,15 @@
 #!/bin/bash
+
+KEXTBUNDLEID="org.vfsforgit.PrjFSKext"
+KEXTPATH="/Library/Extensions/PrjFSKext.kext"
+isKextLoadedCmd="/usr/sbin/kextstat -l -b $KEXTBUNDLEID | wc -l"
+isKextLoaded=$(eval $isKextLoadedCmd)
+if [ "$isKextLoaded" -eq 0 ]; then
```

Comment:
Are there any error conditions we might have to account for here?

Reply:
`kextstat` can return console output which can indicate couple things
- macOS successfully auto-loaded our kext after install. 
- there was a `kextstat` error.

Updated `postflight` to log`kextstat` output, in case it was a `kextstat` error.